모의고사4/7465.py

```diff
@@ -0,0 +1,30 @@
+def DFS(start):
+    if check[start] == False :
+      stack = [start]
+      check[start] = True
+
+      while stack:
+        cur = stack.pop()
+        for adj in l[cur]:
+          if not check[adj]:
+            check[adj] = True
+            stack.append(adj)
+      else:
+        global cnt
+        cnt += 1
```

Comment:
깔끔한 풀이네요! 👍
`while` - `else` 구문은 **while이 break로 종료되지 않았을 때 else를 실행**하라는 뜻입니다! 즉, 지금의 코드에선 `break`가 없으므로 무조건 `else` 문이 실행됩니다. 따라서 아래와 같이 바꿔볼 수 있을 것 같아요! 😄 
```suggestion
      while stack:
        cur = stack.pop()
        for adj in l[cur]:
          if not check[adj]:
            check[adj] = True
            stack.append(adj)
      global cnt
      cnt += 1
```